PowerPointLabs/PowerPointLabs/ColorsLab/ColorsLabPaneWPF.xaml.cs

```diff
@@ -693,7 +693,7 @@ void LeftMouseButtonUpEventHandler()
             if (_eyedropperMode == MODE.MAIN)
             {
                 selectedColorRectangle.Opacity = 1;
-                if (timer1Ticks > CLICK_THRESHOLD)
+                if (timer1Ticks >= CLICK_THRESHOLD)
```

Comment:
is there a reason for changing it to from `>` to `>=` here and then `>=` to `>` below?

Reply:
On line 716, the function displays a notification if the timer is below the click threshold. If the user releases the mouse (mouseupevent is called) when click threshold is exactly 2, the color won't be picked and there won't be a notification either. This might confuse the user.